packages/preview/basic-report/0.3.1/README.md

```diff
@@ -0,0 +1,67 @@
+# Basic Report
+
+A [Typst](https://typst.app/home/) template for shorter non-fiction documents like reports, manuals, requirements documentation, student assignments etc. An example document (PDF) that shows how it looks like, can be found [here](https://github.com/roland-KA/basic-report-typst-template/blob/main/examples/main.pdf).
+
+The template comes without bells and whistles and consists just of
+- a titlepage with
+    - logo
+    - title & document category
+    - info block (date, author, affiliation)
+- the TOC 
+- and then the contents (with currently three levels of headings)
+
+The TOC uses roman page numbers, the rest of the document arabic numbers.
+
+For documents with only a few pages there is often no need for an outline and a separate title page. For these documents there is a `compact-mode`, where title information is shown on the first text page. An example for this mode can be found [here](https://github.com/roland-KA/basic-report-typst-template/blob/main/examples/mainCompact.pdf)
+
+## Usage
+
+You can use this template in the Typst web app by clicking "Start from template" on the dashboard and searching for `basic-report`.
+
+Alternatively, you can use the CLI to kick this project off using the command
+
+```shell
+typst init @preview/basic-report
+```
+
+Typst will create a new directory with all the files needed to get you started.
+
+## Fonts
+
+ _**Vollkorn**_ is used for the body text. It's a high quality font for long texts designed by [Friedrich Althausen](http://friedrichalthausen.de/). It was one of the first Google Fonts. But I recommend to download it from the designers [project web page](http://vollkorn-typeface.com/), because that variant has the small caps font included (on [Google Fonts](https://fonts.google.com/specimen/Vollkorn) that's a separate font). The online version of Typst hast Vollkorn preinstalled.
```

Comment:
```suggestion
 _**Vollkorn**_ is used for the body text. It's a high quality font for long texts designed by [Friedrich Althausen](http://friedrichalthausen.de/). It was one of the first Google Fonts. But I recommend to download it from the designers [project web page](http://vollkorn-typeface.com/), because that variant has the small caps font included (on [Google Fonts](https://fonts.google.com/specimen/Vollkorn) that's a separate font). The online version of Typst has Vollkorn preinstalled.
```